lib/extensions/permissions.js

```diff
@@ -102,21 +104,37 @@ async function setAccess (bundleId, permissionsMapping) {
  * @param {string} serviceName - the name of the service. Should be one of
  * `SERVICES` keys.
  * @param {string} simDataRoot - the path to Simulator `data` root
- * @returns {string} - The current status: yes/no/unset
+ * @returns {string} - The current status: yes/no/unset/limited
```

Comment:
the documentation in xcuitest-driver should be updated after this change is merged

Reply:
yes